tests/run/i19178.check

```diff
@@ -0,0 +1,4 @@
+Failure(java.util.NoSuchElementException: enum Foo has no case with ordinal: 3)
+Failure(java.lang.IllegalArgumentException: enum Foo has no case with name: Bar)
```

Comment:
@bishabosha why do those methods throw different exceptions? It seems that both could throw a `NoSuchElementException`.

Reply:
Probably compatibility with Java